### ISSUE TITLE
Livestock exposure

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9942]
**Preferred term label:**
Livestock exposure

**Synonyms**
Horses, cattle, donkeys, sheep

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Glanders

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000528
- label: Livestock exposure
- parents: [Exposure history (HP:4000110)](https://hpo.jax.org/app/browse/term/HP:4000110)
- pmid: 
- def: A medical history of having been exposed to livestock (animals such as cattle and sheep which are kept on a farm ).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9942
- synonyms:
